Std/Tactic/Alias.lean

```diff
@@ -70,10 +79,11 @@ elab (name := alias) mods:declModifiers "alias " alias:ident " := " name:ident :
     let resolved ← resolveGlobalConstNoOverloadWithInfo name
     let const ← getConstInfo resolved
     let declMods ← elabModifiers mods
-    let declMods ← pure { declMods with
-        isNoncomputable := declMods.isNoncomputable || isNoncomputable (← getEnv) const.name
-        isUnsafe := declMods.isUnsafe || const.isUnsafe
-      }
+    let declMods := { declMods with
+      isNoncomputable := declMods.isNoncomputable || isNoncomputable (← getEnv) const.name
+      isUnsafe := declMods.isUnsafe || const.isUnsafe
+      attrs := setDeprecatedTarget resolved declMods.attrs
```

Comment:
`resolved` is used here whereas `const.name` is used two lines before. They must be equal but it's better to use just one to avoid confusion. I have no particular preference which but I guess I did prefer `const.name` when I wrote the previous line :-)